GoMain/src/main/main.go

```diff
@@ -182,11 +187,8 @@ func orchestrationInit() error {
 	restEdgeRouter.Add(ehandle)
 
 	restEdgeRouter.Start()
-
-	if _, err := os.Stat(dataStorageFilePath); err == nil {
-		sd := storagedriver.StorageDriver{}
-		go startup.Bootstrap(dataStorageService, device.Version, &sd)
-	}
+	storageIns := internalapi.GetStorageInstance()
```

Comment:
If the configuration.toml file is not in dataStorageFilePath like #159, is it working?